code/modules/clothing/under/accessories/accessory.dm

```diff
@@ -40,7 +40,7 @@
 		return
 	has_suit.overlays -= inv_overlay
 	has_suit = null
-	src.loc = get_turf(user || src)
+	src.forceMove(get_turf(user || src))
```

Comment:
DM ref is false.

It returns whatever made the or pass, so if you do `null || "true"`, you get `"true"`.

DM is not the only language to do this, Python and Javascript both do it too.